### PR TITLE
Add support for Zeppelin 0.7.0

### PR DESCRIPTION
**Pull request by deanwampler**
New `repo/packages/Z/zeppelin/4` directory. Requires a new `mesosphere/zeppelin:0.7.0`, as in this PR: https://github.com/mesosphere/dcos-zeppelin/pull/15

### CODE REVIEW

**Comment by vishnu2kmohan** [comment]
Please bump Spark to `1.0.9-2.1.0-1-hadoop-2.6`
Ref: https://hub.docker.com/r/mesosphere/spark/tags/

**Comment by vishnu2kmohan** [comment]
Also, I don't actually see the new Zeppelin Docker image tag: `mesosphere/zeppelin:0.7.0` at https://hub.docker.com/r/mesosphere/zeppelin/tags/ yet?

**Comment by deanwampler** [comment]
I just pushed an update for the Spark tag. 
The Zeppelin 0.7.0 image requires this PR: https://github.com/mesosphere/dcos-zeppelin/pull/15.

**Comment by ryadav88** [comment]
@deanwampler LGTM apart from the Zeppelin Docker  image tag 0.7.0.
Will merge when https://github.com/mesosphere/dcos-zeppelin/pull/15 gets merged.

**Comment by deanwampler** [comment]
Awesome. Thanks!

**Comment by jshenguru** [comment]
@deanwampler The package/package.json and package/resource.json will be base64 encoded into the DCOS_PACKAGE_METADATA for DC/OS (specifically the Marathon) to display info. The package/marathon.json.mustache determines how Marathon will run container and what environment variables to set for the container. The package/config.json is what you can set through the Marathon package setup interface, such as docker image for Zeppelin service, environment variables you want Marathon passes into container. Therefore, the pcakge/config.json and the package/marathon.json.mustache come hand in hand.
For zeppelin 0.7.0, you can add the "default": "mesosphere/spark:1.0.9-2.1.0-1-hadoop-2.7" to the package/config.json under the "executor_docker_image" section and the "default": "https://downloads.mesosphere.com/spark/assets/spark-2.1.0-bin-2.7.tgz" to the package/config.json under the "spark" -> "properties" -> "uri" section.

**Comment by deanwampler** [comment]
I just pushed an update with those changes, @jshenguru.

**Comment by jshenguru** [comment]
I forked the dcos-zeppelin project (https://github.com/jshenguru/dcos-zeppelin) and change from package-based to Marathon app based manner and made a zeppelin 0.7.0 docker image (https://hub.docker.com/r/jshenguru/zeppelin/). There is no need for wait anymore.

**Comment by jshenguru** [comment]
image

**Comment by jshenguru** [comment]
I looked into the Mesosphere official Spark docker images and "stole" the following two lines from the conf/spark-env.sh into the zeppelin-env.sh.

[ -f "${MESOS_SANDBOX}/hdfs-site.xml" ] && cp "${MESOS_SANDBOX}/hdfs-site.xml" "${HADOOP_CONF_DIR}"
[ -f "${MESOS_SANDBOX}/core-site.xml" ] && cp "${MESOS_SANDBOX}/core-site.xml" "${HADOOP_CONF_DIR}"

That way, if you have HDFS deployed on your DC/OS, this will grab proper config files that are generated by DC/OS. If you don't have HDFS deployed, nothing gets into the /etc/hadoop (this is what I need for my case).

I've fully updated my Github at (https://github.com/jshenguru/dcos-zeppelin), rebuilt the dcos-zeppelin docker image (I decided to change the name to dcos-zeppelin to avoid confusion) and uploaded to the Docker hub (https://hub.docker.com/r/jshenguru/dcos-zeppelin/).

I did a quick test using the tutorial code and everything works fine.

I did this because I don't know how long it will take to get all these changes merged into the Mesosphere official repository.

**Comment by ryadav88** [comment]
@deanwampler do you want to update the package to use the image that @jshenguru created?

**Comment by tobilg** [comment]
Is there any progress here @jshenguru @deanwampler? What needs to be done to finalize this?
cc @ryadav88

**Comment by deanwampler** [comment]
I just pushed an update to use @jshenguru's docker image for Zeppelin and use the latest mesosphere Spark plugin.

File: repo/packages/Z/zeppelin/4/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "tags": [
+        "nflabs",
+        "framework",
+        "bigdata",
+        "spark",
+        "flink",
+        "notebook",
+        "interactive"
+    ],
+    "website": "https://github.com/mesosphere/dcos-zeppelin",
```
Comment (ryadav88) [inline repo/packages/Z/zeppelin/4/package.json]: Please add `preInstallNotes` with this line ```This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production!```

File: repo/packages/Z/zeppelin/4/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "packagingVersion": "2.0",
```
Comment (ryadav88) [inline repo/packages/Z/zeppelin/4/package.json]: change `packagingVersion` to 3.0

**Comment by deanwampler** [comment]
I just pushed an update for both changes.

**Pull request merged**